INN/umbrella-citylimits
# Special projects "featured content area" widget

## Changes

This pull request makes the following changes:

- Created the `City Limits Special Projects Featured Content` widget to display the rest of the posts on a series landing page, loosely based off of Largo Recent Posts widget

Desktop:
![screencapture-citylimits-test-series-election-watch-2018-2019-11-08-14_03_55](https://user-images.githubusercontent.com/18353636/68503411-c3823e80-0230-11ea-8c55-ec1619e6dc2f.png)

Mobile:
![screencapture-citylimits-test-series-election-watch-2018-2019-11-08-14_04_34](https://user-images.githubusercontent.com/18353636/68503402-c11fe480-0230-11ea-9af8-22d0fc90bf6b.png)

## Why



For #73 

## Testing/Questions

Features that this PR affects:

- Special projects series posts being displayed


Questions that need to be answered before merging:

- [ ] The widget is named `City Limits Special Projects Featured Content` since that's what it's named in #73, but that feels like it's kind of misleading since this is really just posts that haven't been displayed already. Should we name it something like `City Limits Special Projects Series Archive`?
- I had to redeclare `largo_load_more_posts_choose_partial` since the filter `largo_lmp_template_partial` doesn't pass the `$query_vars` for me to be able to check if the widget is calling LMP. Is there an easier way to do this besides redeclaring this function?
- [ ] Did I miss any edge cases?
- [ ] Does it work as expected?

Steps to test this PR:

1. Find a series with more than ( 7 (7 series widget) + 8 (this widget) = ) 15 posts
2. Add the new `City Limits Seven Series Posts` widget to the landing page and set it to avoid duplicates
3. Add the `City Limits Special Projects Featured Content` widget to a landing page and make sure it's set to avoid duplicate posts
4. Make sure both widgets do not display duplicate posts
5. Find any weird edge cases
